### DISCUSSION TITLE
현존 최고의 LPR 인거 같습니다. 잘 사용하고 있는데, 프로그램 발전을 위해 몇 사례 알려드립니다.

### DISCUSSION THREAD

**Discussion by Lawmedic** [bobhyun/TS-ANPR#3] [Q&A]
.

**Comment by bobhyun** [comment]
안녕하세요.

**사람이 읽을 수 있는 건 나도 읽을 수 있어!** 를 모토로 하고 있지만 아직 부족한 점이 많네요.😢
그래도 고객님들이 보내주시는 희귀한 현장 데이터들이 인식률 개선에 많은 도움이 되고 있어
머지않아 사람을 능가하는 수준에 도달할 수 있다는 희망을 가지게 됩니다.
보내주신 자료 감사드리며 인식이 잘 안되는 부분은 개선해서 다음 버전에 적용하겠습니다.

🙏🏽 **자료 업로드 관련하여 당부의 말씀을 드립니다.**
- *타인의 차량 번호판 사진을 동의없이 인터넷 상에 공개할 경우 개인 정보 침해의 소지가 있을 수 있으니 
출처를 명시하거나 한 글자 이상 모자이크 처리해서 올려 주시길 부탁드립니다.*
- *공개할 목적이 아닌 경우는 제 이메일(bobhyun@gmail.com)로 직접 보내주시면 고맙겠습니다.*
